pkg/util/http2.go

```diff
@@ -0,0 +1,82 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"golang.org/x/net/http2"
+)
+
+func DoHTTP2Get(methodName string, ip string, port int, certFile, keyFile, caFile string, insecure bool, timeout int) ([]byte, error) {
+
+	httpClient := &http.Client{
```

Comment:
The initialization logic of the client should be independent, and it is not necessary to do repeated things every time an HTTP request is made.

Reply:
We can hide the difference between http/1.x and http/2

Reply:
I think cce is not standard implementation, maybe there are another implementation in the future. In order to make the code logic easy to manage, we'd better implement it separately.

Reply:
There is indeed a problem with the implementation in `pkg/util/http.go` in the existing code, but I think you can directly encapsulate a general HTTP Client in this PR to facilitate refactoring. In addition, I understand that this PR has nothing to do with whether the CCE implementation is standard or not.

Reply:
OK, I will rewrite this part.